packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -338,5 +338,58 @@ void main() {
             excludes: <Offset>[const Offset(16.1, -40.0), const Offset(-98.1, -40.0)],
           ));
     await gesture.up();
+
+    // Test that the neck stretches when the text scale gets smaller.
+    await tester.pumpWidget(buildApp('1000000', sliderValue: 0.0, textScale: 0.5));
+    center = tester.getCenter(find.byType(Slider));
+    gesture = await tester.startGesture(center);
+    await tester.pump();
```

Comment:
Would it be simpler to just use `await tester.pumpAndSettle()` here (and below)?

Reply:
Yes.  Done.